ospf6d/ospf6_interface.c

```diff
@@ -1287,7 +1287,8 @@ DEFUN (no_auto_cost_reference_bandwidth,
 
 DEFUN (ipv6_ospf6_hellointerval,
        ipv6_ospf6_hellointerval_cmd,
-       "ipv6 ospf6 hello-interval (1-65535)",
+       "[no] ipv6 ospf6 hello-interval (1-65535)",
```

Comment:
shouldn't the integer parameter be optional for the "no" form?

and yeah i know that means you have to duplicate the entire command ;)